reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py

```diff
@@ -0,0 +1,79 @@
+
+import sys
+import json
+import pprint
+
+
+class EEDefs:
+    NAME = "name"
+    ID = "id"
+    CREATED = "created"
+    TEST_PIPELINE_ID = "testPipelineId"
```

Comment:
the `test` variables are actual keywords? or are they testing ones?

Reply:
yes these are keyworks in EE definition, it refers to pipeline, MLApp use to test this EE